<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Moravecui</class>
 <widget class="QWidget" name="Moravecui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moravecui</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="17" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>卷积核长</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>保存路径</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>二次卷积核</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>阈值</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>提取图像</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>卷积核宽</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="17" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>算子</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>Moravecui</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Moravecui</receiver>
   <slot>yunsuan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Moravecui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Moravecui</receiver>
   <slot>suanzi(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Moravecui</receiver>
   <slot>huofu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>yunsuan()</slot>
  <slot>suanzi(QString)</slot>
  <slot>huofu()</slot>
 </slots>
</ui>
